scripts/lint-examples/package.json

```diff
@@ -20,7 +20,7 @@
     "glob-promise": "^4.2.2",
     "prettier": "2.8.8",
     "react": "18.2.0",
-    "react-native": "^0.73.0",
+    "react-native": "^0.74.1",
```

Comment:
Please update the other dependencies here as well. See https://react-native-community.github.io/upgrade-helper/?from=0.73.0&to=0.76.0-rc.2#RnDiffApp-package.json for the versions which have changed.

You will also need to run `yarn install` to update the lockfile and check that in.

Reply:
I did... but someone added a post-commit hook that broke it... https://github.com/facebook/react-native-website/blob/888a1f590a1a44389fe6d46b96b1893a86e9659c/.github/workflows/pre-merge.yml#L19